### DISCUSSION TITLE
社区推荐用什么方式同步Mysql数据（增量 和 全量）？

### DISCUSSION THREAD

**Discussion by yclxiao** [apache/doris#14144] [A - General / Q&A]
社区推荐用什么方式同步Mysql数据（增量 和 全量）

**Comment by zy-kkk** [comment]
推荐Flink CDC方式https://mp.weixin.qq.com/s/ejDChH5ScUK4pSxo2EH_4w

**Reply by yclxiao** [reply]
好的，谢谢！
那全量同步mysql数据 推荐用什么方式

**Reply by caoliang-web** [reply]
全量的话，可以使用datax-doriswriter : https://doris.apache.org/zh-CN/docs/dev/ecosystem/datax?_highlight=datax
